writer/stdout.go

```diff
@@ -11,6 +11,6 @@ func NewStdoutWriter() *StdoutWriter {
 }
 
 func (s *StdoutWriter) Write(_path string, data []byte) error {
-	log.Print(data)
```

Comment:
ah nice catch, guess that was useless before haha

Reply:
haha, yeah it happened while debugging and I used `-print` just to be presented with lots of numbers! :stuck_out_tongue: